content/source/docs/cloud/api/ssh-keys.html.md

```diff
@@ -26,7 +26,7 @@ The `ssh-key` object represents an SSH key which includes a name and the SSH pri
 
 SSH keys can be used in two places:
 
-- They can be assigned to VCS provider integrations ([available in the API as `oauth-tokens`](./oauth-tokens.html)). Bitbucket Server requires an SSH key; other providers only need an SSH key if your repositories include submodules that are only accessible via SSH (instead of your VCS provider's API).
+- They can be assigned to VCS provider integrations ([available in the API as `oauth-tokens`](./oauth-tokens.html)). Azure DevOps Server and Bitbucket Server require an SSH key. Other providers only need an SSH key if your repositories include submodules that are only accessible via SSH (instead of your VCS provider's API).
```

Comment:
Here it says that these require SSH keys, but on the BitBucket and Azure DevOps Server pages the SSH keys are listed as optional.